pkg/dashboards/dashboards_api.go

```diff
@@ -2000,15 +2146,6 @@ func (a *Dashboards) DashboardDeleteWithContext(
 		return nil, err
 	}
 
-	// If we got errors back, wrap them all up
-	if len(resp.DashboardDeleteResult.Errors) > 0 {
-		errs := fmt.Errorf("query error")
-		for _, err := range resp.DashboardDeleteResult.Errors {
-			errs = fmt.Errorf("%w; %s", errs, err.Description)
-		}
-		return nil, errs
-	}
-
```

Comment:
Can you please revert this change? It looks like this portion of the code was manually added (and not via Tutone) in order to ensure effective error handling, which is why code generation via Tutone would suggest removing these changes while we might actually need them.

https://github.com/newrelic/newrelic-client-go/commit/8f3b6ddebf46210250b6d4db36ce72763920d07d

Reply:
same here